src/programs/marinade-referral-program.ts

```diff
@@ -73,6 +74,7 @@ export class MarinadeReferralProgram {
     transferFrom,
     systemProgram: SYSTEM_PROGRAM_ID,
     tokenProgram: TOKEN_PROGRAM_ID,
+    tokenPartnerAccount: assertNotNullAndReturn(this.tokenPartnerAccount, "The referral partner account must be provided!"),
```

Comment:
"the referral partnet account" is a SOL Native account, base for the ATA mSOL-token-partner-account
Maybe we should rename "tokenPartnerAccount" to "mSOLTokenPartnerAccount" to be more precise and differentiate from "partnerAccount" that's a Native SOL account, owner of the ATA